public/views/dashboard.html

```diff
@@ -1,4 +1,196 @@
+                    <th class="lost">Games Lost</th>
+                    <th class="donation">Donation</th>
+                  </tr>
+                </thead>
+                <tbody>
+                  <tr ng-repeat="stat in stats | limitTo:8">
+                    <td>{{stat.date}}</td>
+                    <td>{{stat.played}}</td>
+                    <td>{{stat.won}}</td>
+                    <td>{{stat.lost}}</td>
+                    <td>{{stat.donation | currency}}</td>
+                  </tr>
+                </tbody>
+              </table>
+              <div ng-show="nextStat()">
+                <ul class="pagination">
+                  <li class="disabled"><a href=""><i class="material-icons">chevron_left</i></a></li>
+                  <li class="active"><a href="">1</a></li>
+                  <li class="waves-effect"><a>2</a></li>
+                  <li class="waves-effect"><a>3</a></li>
+                  <li class="waves-effect"><a><i class="material-icons">chevron_right</i></a></li>
+                </ul>
+              </div>
+            </section>
+  
+  
+          </aside>
+        </div>
+      </div>
+      <!-- Donate -->
+      <section class="cfh-section purple darken-2" id="donate">
+        <div class="container">
+          <div class="row">
+            <div class="col s12 m4 hide-on-small-only non-profit">
+              <img class="ribbon" src="../img/nonprofit_ribbon.png" />
+            </div>
+            <div class="col s12 m8 authUser">
+              <h4>Support the cause</h4>
+              <p>
+                To unlock the community’s incredible giving potential, we've integrated donation widgets ( via <a class="red-text " target="_blank" href="https://crowdrise.com"> crowdrise.com </a> ), along with some not-so-subtle reminders throughout the
+                game to give you the opportunity to donate. If you're signed in, and when you donate, we will upgrade your account, award you value points and place a crown on your avatar's head (to remind other players how much better you are than them).
+              </p>
+              <br>
+              <a class="btn brand-btn brand-btn-donate-invert btn-large" href="https://www.crowdrise.com/donate/project/cfhio/cards4humanity?widget=true&redirect_to=http%3A%2F%2Flocalhost%3A3000%2F%23!%2F&stylesheet=&amounts="> 
+                    Donate
+                  </a>
+            </div>
+          </div>
+          <br>
+          <p class="giving-quote white-text center">"If you don't give back, no one will like you&trade;" -
+            <a class="red-text" target="_blank" href="https://crowdrise.com"> crowdrise.com </a>
+          </p>
+        </div>
+      </section>
+      <!-- The team -->
+      <footer class="cfh-section section scrollspy" id="partners">
+          <nav class="container">
+            <div class="left">
+                <a href="#home" class="cfh-logo black-text"><img src="img/cfh-logo.png" alt="cfh-logo"/></a>
```

Comment:
Why is the image path different from other implementation?

Reply:
That was an oversight. Resolved!